api/routes/users.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import APIRouter
+from pydantic import BaseModel
+
+router = APIRouter()
+users = {}
+
+
+def email_duplicates(email: str):
+    users_values = list(users.values())
+    for user in users_values:
+        if user.email == email:
+            return True
+    return False
+
+
+class User(BaseModel):
+    first_name: str
+    last_name: str
+    email: str
+
+
+@router.post("/create-user/{user_id}")
```

Comment:
@allar14 User_id needs to be generated automatically. You generally do not pass id as a parameter when you are creating new entity record.

Reply:
@allar14 , you can take a look at how the POST method is done for /trackers router

Reply:
updated the code addressing your comments above